compute/baremetal.py

```diff
@@ -145,3 +145,18 @@ def role(self, roles: list) -> None:
     def shortname(self) -> str:
         """Return the shortform of the hostname."""
         return self.hostname.split(".")[0]
+
+    @property
+    def no_of_lvms(self) -> int:
+        """Return the number of lvms to be created."""
+        return self.params.get("no-of-lvms")
+
+    @property
+    def lvm_size(self) -> int:
+        """Return the lvm size to be created."""
+        return self.params.get("lvm_size")
```

Comment:
On second thoughts, let's not have them part of the `CephNode`... as they are more like configurations that need to take place... then we can update the `volumes` property with the paths to the lvm's that were created.

Reply:
Providing config via test case or overrides.